docs/cis_self_assessment.md

```diff
@@ -1367,10 +1247,19 @@ Ensure that the API Server only makes use of Strong Cryptographic Ciphers (Not S
 TLS ciphers have had a number of known vulnerabilities and weaknesses, which can reduce the protection provided by them. By default Kubernetes supports a number of TLS ciphersuites including some that have security concerns, weakening the protection provided.
 </details>
 
-**Result:** Pass
+**Result:** **Not Scored - Operator Dependent**
 
-**Remediation:**
+**Audit:**
+Run the below command on the master node.
 
+```bash
+/bin/ps -ef | grep kube-apiserver | grep -v grep
+```
+
+Verify that the `--tls-cipher-suites` argument is set as outlined in the remediation procedure below.
```

Comment:
We could possibly "do better" and set this flag as the upstread CIS doc outlines. But that is not a blocker. Can you open a separate issue for implementing this, but leave this doc as-is for now, as what we are doing is acceptable for ga